src/Commands/RunScan.ts

```diff
@@ -78,12 +76,12 @@ export class RunScan implements CommandModule {
           'A list of specific urls that should be included into crawler.'
       })
       .option('test', {
-        choices: Helpers.toArray(TestType),
-        defaultDescription: `[${SCAN_TESTS_TO_RUN_BY_DEFAULT.map(
-          (item) => `"${item}"`
-        ).join(',')}]`,
         array: true,
-        describe: 'A list of tests which you want to run during a scan.'
+        describe:
+          'A list of tests which you want to run during a scan.' +
+          'Default tests are run if not set any' +
+          'See default tests in a documentation: https://docs.brightsec.com/docs/running-a-scan' +
+          'and all the available tests https://docs.brightsec.com/docs/vulnerability-guide'
```

Comment:
It looks like the wording has been changed https://github.com/NeuraLegion/bright-cli/pull/554#discussion_r1665816842

```suggestion
          'A list of tests to run during a scan. ' +
          'If no tests are specified, the default tests will be run.' +
          'For more information on the default tests, refer to the documentation: https://docs.brightsec.com/docs/running-a-scan' +
          'Additional details about available tests can be found here: ' +
          'https://docs.brightsec.com/docs/vulnerability-guide'
```